react/week1/todo-app/src/App.js

```diff
@@ -0,0 +1,81 @@
+import logo from './logo.svg';
+import './App.css';
+
+function DateItem(props) {
+  let d = new Date();
+  d.setFullYear(props.date.year, props.date.month - 1, props.date.day);
+  const today = new Date();
+  const validDate = (d > today) ? 'valid': 'expired';
```

Comment:
This is fine but try to give your variables names that are easier to understand. So for example instead of `let d` you could use `let deadlineDate`. A small challenge if you want to: can you avoid using `let` here?